### ISSUE TITLE
Unknown "see_more" filter.

### ISSUE THREAD

**Issue by chrisvidal** [responsiv/pyrolancer-plugin#3]
After signin with a freelancer user, i tried to ask a question to a job offer, and got this error:

```
Unknown "see_more" filter.
/Users/abcdef/Sites/oc-pyro/themes/responsiv-freelance/partials/project/message.htm line 71

{% endmacro %}
  
{# View the message #}
{% macro render_message_content(message) %}
    <div class="text">
  
        {{ message.content_html|see_more }}
  
        {% if message.created_at != message.updated_at %}
            <small class="text-muted">Last updated {{ message.updated_at.diffForHumans }}</small>
        {% endif %}
  
    </div>

```

**Comment by chrisvidal** [comment]
any updates please?

**Comment by daftspunk** [comment]
This should be fixed in v1.0.5. Thanks for the report 

Commit:  42dc2ef
